src/raftstore/store/peer.rs

```diff
@@ -1337,13 +1337,17 @@ impl Peer {
 
 pub fn check_epoch(region: &metapb::Region, req: &RaftCmdRequest) -> Result<()> {
     let (mut check_ver, mut check_conf_ver) = (false, false);
+    let mut left_derive = false;
```

Comment:
I guess this can only check Admin Split command, for other commands, we also can meet stale problems.